examples/stubbing-spying__intercept/cypress/integration/glob-matching-intercept-url.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('intercept', () => {
+    cy.intercept('/api/roles')
+    // will match https://example.com/api/roles
+    // will not match https://example.com/api/roles?country=US
+    
+    cy.intercept('/api/roles*')
+    // will match https://example.com/api/roles
+    // will match https://example.com/api/roles?country=US
+    
+    cy.intercept('/api/roles?*utm_source=+(My%App|https*)')
```

Comment:
Is `My%App` correct? I'd expect to see either `My App` or `My%20App` here (and in the matched URLs). Perhaps `MyApp` would be better.

Reply:
Done